recipes/scotch/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd src/
+echo 'prefix = $PREFIX' > Makefile.inc
+echo '' >> Makefile.inc
+if [ `uname` == "Darwin" ]; then
+    cp Make.inc/Makefile.inc.i686_mac_darwin10 Makefile.inc
+    sed -i '' 's/-DSCOTCH_PTHREAD//g' Makefile.inc
+    sed -i '' 's/-O3/-O3 -fPIC/g' Makefile.inc
+else
+  cp Make.inc/Makefile.inc.x86-64_pc_linux2 Makefile.inc
+  sed -i "s@CFLAGS\t\t=@CFLAGS\t= -I${PREFIX}/include@" Makefile.inc
```

Comment:
Why would be to pick up `zlib`.

How would be to add `-L${PREFIX}/lib`. Though I really am hoping they have a configure file we overlooked.